emitter/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "compilerOptions": {
+        "noCheck": true
```

Comment:
Then what is the reason for having TS?

Reply:
to allow write TS but we don't want the compiler to give any hints